### PR TITLE
change set-output to the new system

### PR DESCRIPTION
**Pull request by max246**
Update the system to set output using the new convention from github

This is releated to ticket #6

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
```
Comment (foosel) [inline package.json]: Please get rid of this, we don't need that, and if we do, I'd like to know why and see the links replaced ;)

**Review by foosel — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (max246) [reply package.json]: I suppose the other one as well from package-lock?

Comment (foosel) [reply package.json]: Yes, indeed 👍

Comment (foosel) [reply package.json]: and also the (accidental) rename of `issue-validation/package-locks.json` to `issue-validation/package-locka.json`

**Review by foosel — APPROVED** [review]

**Pull request merged**